packages/sdks/snippets/react/src/routes/blueprints/NavLinks.tsx

```diff
@@ -0,0 +1,32 @@
+import { fetchOneEntry, type BuilderContent } from '@builder.io/sdk-react';
+import { useEffect, useState } from 'react';
+import { NavBar } from '../../components/NavBar';
+
+export function NavLinks() {
+  const [links, setLinks] = useState<BuilderContent | null>(null);
+
+  useEffect(() => {
+    fetchOneEntry({
+      model: 'navigation-links',
+      apiKey: 'ee9f13b4981e489a9a1209887695ef2b',
+    }).then((data) => {
+      setLinks(data);
+    });
+  }, []);
+
+  return (
+    <nav>
+      {/* Brand name */}
+      <h1>Acme company</h1>
+
+      {/* Navigation links Builder Component */}
+      <NavBar links={links} />
```

Comment:
are we sure to update the structure of the code to introduce a new component? why can't we inline the  `<nav>` here? https://www.builder.io/c/blueprints/navigation-links Is this the correct link that I should check?

Reply:
oh i see, we already have it in that way for angular snippets. but still a q for @uttej-vsk and @samijaber